### PR TITLE
feat: add resource create/delete actions

### PR DESCRIPTION
**Pull request by varnastadeus**

### CODE REVIEW

File: actions/create_handler.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	if newObj.GetNamespace() == "" {
+		return errors.New("object namespace is missing")
+	}
+
+	log := h.log.WithFields(logrus.Fields{
+		"id":       action.ID,
+		"action":   reflect.TypeOf(action.Data()).String(),
+		"group":    req.Group,
+		"resource": req.Resource,
+		"version":  req.Version,
```
Comment (mjudeikis) [inline actions/create_handler.go:53]: I would construct proper GVR resource as single field. Easier to read in logs...

**Review by mjudeikis — APPROVED** [review]
I would just tune logging as GRV is not quite intended to be separate fields in real world... everybody is used to apps/v1 or `stuff.core/v1alpha1`

File: actions/create_handler.go (hunk 1 above)
Comment (varnastadeus) [reply actions/create_handler.go:53]: makes sense, had similar idea initially but kept it consistent with `patch` handler. Will change it there as well 🙏

File: actions/create_handler.go [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+			return err
+		}
+
+		patch, err := jsonpatch.CreateMergePatch(original, modified)
+		if err != nil {
+			return fmt.Errorf("creating patch: %w", err)
+		}
+
+		// If resources are identical, patch will be equal '{}'.
+		if len(patch) <= 2 {
```
Comment (UndeadRat22) [inline actions/create_handler.go:103]: if it will be equal to {}, we can just say if patch == '{}' return nil

**Review by UndeadRat22 — APPROVED** [review]

**Pull request merged**